functions-and-scope.js

```diff
@@ -17,6 +17,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 6
 
 
+let numberOfCumLaude = 0;
+for (let i = 0; i < grades.length; i++) {
+
+
+    if (grades[i] >= 8) {
+        numberOfCumLaude +=1;
```

Comment:
Je kan deze nog verder versimpelen:
```suggestion
        numberOfCumLaude++;
```